### PR TITLE
1950 sidenav tests

### PR DESCRIPTION
**Pull request by AlexSofft**

### CODE REVIEW

**Comment by mf-code** [comment]
[6091] - cBTn4ajMrX13

**Comment by mf-code** [comment]
[6091] - fVt42jTZ2rQO

**Comment by mf-code** [comment]
[6091] - xpEvCxED5sRO

**Comment by mf-code** [comment]
[6091] - VSCcWvwZnlo7

**Comment by mf-code** [comment]
[6091] - zxC9JnYLWPNv

**Comment by mf-code** [comment]
[6091] - M1gJj0aHz2Y6

**Comment by mf-code** [comment]
[6091] - VLdzmqU5u3y4

**Comment by mf-code** [comment]
[6091] - Y8oyt7XQKyTE

**Comment by mf-code** [comment]
[6091] - FQ1dGC9bvuGx

**Comment by mf-code** [comment]
[6091] - qLWC4MWBmmS1

**Comment by mf-code** [comment]
[6091] - 8WxlRCdQWZOu

**Comment by mf-code** [comment]
[6091] - yH3oMASPAPe4

**Comment by mf-code** [comment]
[6091] - a73aa49hibw6

**Comment by mf-code** [comment]
[6091] - SYvHzQ8A7Vpt

**Comment by mf-code** [comment]
[6091] - s1VmB0MYZV6G

**Comment by mf-code** [comment]
[6091] - gGQ1dtXOhEAN

**Comment by mf-code** [comment]
[6091] - qqShgnKPhNtL

**Comment by mf-code** [comment]
[6091] - oWepcKA9wwWK

**Comment by mf-code** [comment]
[6091] - openjdk8 - Allure report on Netlify: https://5f5f2347da13e38ccccccf84--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[6091] - openjdk10 - Allure report on Netlify: https://5f5f240da21a971985b9966c--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[6096] - tuOCI609Eubx

**Comment by mf-code** [comment]
[6096] - Cwdrthxi3XRv

**Comment by mf-code** [comment]
[6096] - ZNjBZqUDLA0w

**Comment by mf-code** [comment]
[6096] - vHrrt5kIBt1E

**Comment by mf-code** [comment]
[6096] - 2FhcRpZFHpFG

**Comment by mf-code** [comment]
[6096] - gjB1Kyq26xes

**Comment by mf-code** [comment]
[6096] - LqvaTuJo6lSz

**Comment by mf-code** [comment]
[6096] - nX6Ev0kSGEmf

**Comment by mf-code** [comment]
[6096] - ncU60AmNDSTr

**Comment by mf-code** [comment]
[6096] - F2SfoVGl9lVF

**Comment by mf-code** [comment]
[6096] - Pg4Tv5mkHxNv

**Comment by mf-code** [comment]
[6096] - bio1fS4GRX3e

**Comment by mf-code** [comment]
[6096] - wwp93jokGOfK

**Comment by mf-code** [comment]
[6096] - GkN40yDQEop3

**Comment by mf-code** [comment]
[6096] - uEUkkxb8c8XB

**Comment by mf-code** [comment]
[6096] - NJhEDTQcPvxt

**Comment by mf-code** [comment]
[6096] - ikYd5CVYSq5W

**Comment by mf-code** [comment]
[6096] - BEAwuH1FrpeK

**Comment by mf-code** [comment]
[6096] - openjdk8 - Allure report on Netlify: https://5f5f5a4ffff3670d9bb9be49--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[6096] - openjdk10 - Allure report on Netlify: https://5f5f5b0ab8c1b218869483bd--musing-swartz-6fc060.netlify.app

File: jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/unit/SideNavUnitTests.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    @Test
+    public void getResponsiveResultsTest() {
+        toolbarToggle.click();
+        responsiveContent.getSideNavLinks().get(1).click();
+        assertEquals(responsiveContent.getResponsiveResults().get(1).getText(), "Selected Nav Item 1");
+    }
+}
+
+//
+//    @JDIAction("Get '{name}' side nav content")
```
Comment (gbuh) [inline jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/unit/SideNavUnitTests.java:55]: Why do we need this commented content here?

**Comment by mf-code** [comment]
[6099] - yztVwUbQ2mz8

**Comment by mf-code** [comment]
[6099] - IzeAueIGXsIa

**Comment by mf-code** [comment]
[6099] - Obm1VdIJUtDk

**Comment by mf-code** [comment]
[6099] - nKPvPogDGmsS

**Comment by mf-code** [comment]
[6099] - utBnJg117C3o

**Comment by mf-code** [comment]
[6099] - MLStPxsQ2cV0

**Comment by mf-code** [comment]
[6099] - S25eNYw5mNMz

**Comment by mf-code** [comment]
[6099] - U5JnYtllk2YH

**Comment by mf-code** [comment]
[6099] - cXKqErNDxFYB

**Comment by mf-code** [comment]
[6099] - ZYmhQwHT3ovX

**Comment by mf-code** [comment]
[6099] - mtYIY7BzSHlo

**Comment by mf-code** [comment]
[6099] - 6OBUSmagBFOr

**Comment by mf-code** [comment]
[6099] - UFIX652FRzJI

**Comment by mf-code** [comment]
[6099] - mhSfSRPIdJ2y

**Comment by mf-code** [comment]
[6099] - DbaXQ1z4OKfN

**Comment by mf-code** [comment]
[6099] - bWr9cWJ2TA5D

**Comment by mf-code** [comment]
[6099] - mMl0BCWGgZLS

**Comment by mf-code** [comment]
[6099] - fMEiduJC6GHU

**Comment by mf-code** [comment]
[6099] - openjdk8 - Allure report on Netlify: https://5f5fc62591a9c44353b8f73e--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[6099] - openjdk10 - Allure report on Netlify: https://5f5fc6fb39d0b14dc40eff69--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[6102] - JvQcry2QmU5T

**Comment by mf-code** [comment]
[6102] - JPEBqNTDi5J7

**Comment by mf-code** [comment]
[6102] - nnDXIQHRYiTp

**Comment by mf-code** [comment]
[6102] - 0fMkfvhL3vy9

**Comment by mf-code** [comment]
[6102] - HrypbBBVtQTu

**Comment by mf-code** [comment]
[6102] - t6cFJOFnn70y

**Comment by mf-code** [comment]
[6102] - QuOVcaJSh16U

**Comment by mf-code** [comment]
[6102] - Ev6DbsN4Hlsb

**Comment by mf-code** [comment]
[6102] - yvcIBY8yveNu

**Comment by mf-code** [comment]
[6102] - Y5H7BsLHVMZc

**Comment by mf-code** [comment]
[6102] - LBHxlF6cWgDT

**Comment by mf-code** [comment]
[6102] - D1v4l8reSCk5

**Comment by mf-code** [comment]
[6102] - 2YdeVL34qDPN

**Comment by mf-code** [comment]
[6102] - w4UYoO8j4LTS

**Comment by mf-code** [comment]
[6102] - X0XDlMUvsb8T

**Comment by mf-code** [comment]
[6102] - z258EebIkwBE

**Comment by mf-code** [comment]
[6102] - s5qgg1RMxr7T

**Comment by mf-code** [comment]
[6102] - GJrBXIgN32rl

**Comment by mf-code** [comment]
[6102] - openjdk8 - Allure report on Netlify: https://5f6079ad1c1274847002dc90--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[6102] - openjdk10 - Allure report on Netlify: https://5f607a79a297b6a47ecbcef7--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[6106] - K2TI2OKwqFbd

**Comment by mf-code** [comment]
[6106] - lzObGmJQs9O0

**Comment by mf-code** [comment]
[6106] - UTBSduUpzC1i

**Comment by mf-code** [comment]
[6106] - j0tFTn3dHx09

**Comment by mf-code** [comment]
[6106] - PVm5rE9cjo3X

**Comment by mf-code** [comment]
[6106] - lsEuO6mz7Whz

**Comment by mf-code** [comment]
[6106] - DxMcS25vkkdt

**Comment by mf-code** [comment]
[6106] - vBwIjyhRg6bb

**Comment by mf-code** [comment]
[6106] - ndNpmIy9ejaA

**Comment by mf-code** [comment]
[6106] - JPFVU3LjFq4v

**Comment by mf-code** [comment]
[6106] - c9AA3HLDSz3A

**Comment by mf-code** [comment]
[6106] - klohQsCtLpcr

**Comment by mf-code** [comment]
[6106] - vx8cqd0fkm4T

**Comment by mf-code** [comment]
[6106] - 8hn9c5hWpXPn

**Comment by mf-code** [comment]
[6106] - iQidChpWze25

**Comment by mf-code** [comment]
[6106] - YX5szUpHofwo

**Comment by mf-code** [comment]
[6106] - Dd89Imc4AN16

**Comment by mf-code** [comment]
[6106] - NmWfOusIrYvV

**Comment by mf-code** [comment]
[6106] - openjdk8 - Allure report on Netlify: https://5f60ac20de9327014fcda8bb--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[6107] - sQtXLhuoE5kv

**Comment by mf-code** [comment]
[6107] - MMEglJtBOTFU

**Comment by mf-code** [comment]
[6107] - 7XoEfY5RRFEb

**Comment by mf-code** [comment]
[6106] - openjdk10 - Allure report on Netlify: https://5f60acd7d1fbe700be7d97cf--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[6107] - 3G4gAfrr7SnD

**Comment by mf-code** [comment]
[6107] - hyzcAcm8fiZX

**Comment by mf-code** [comment]
[6107] - in7rTjZlz7dw

**Comment by mf-code** [comment]
[6107] - lYKTrkl353zc

**Comment by mf-code** [comment]
[6107] - TgSnkGcx8n03

**Comment by mf-code** [comment]
[6107] - PdkW7d5PmIpZ

**Comment by mf-code** [comment]
[6107] - 1OJqNGfOL9bE

**Comment by mf-code** [comment]
[6107] - 9WX19mBdZvkF

**Comment by mf-code** [comment]
[6107] - gcpi8WE3cXad

**Comment by mf-code** [comment]
[6107] - UZWlKrQRsTRo

**Comment by mf-code** [comment]
[6107] - VYSg2M5mEn98

**Comment by mf-code** [comment]
[6107] - fKpAKARcmco5

**Comment by mf-code** [comment]
[6108] - eNm3xzajCw2l

**Comment by mf-code** [comment]
[6108] - yqFspe0AAmMN

**Comment by mf-code** [comment]
[6108] - R6hh4HKDlnDY

**Comment by mf-code** [comment]
[6108] -

**Comment by mf-code** [comment]
[6107] - DxX7E5TVfKAU

**Comment by mf-code** [comment]
[6108] - k0LoEr4xN0iO

**Comment by mf-code** [comment]
[6108] - sxbbSDC5tjLh

**Comment by mf-code** [comment]
[6108] - eVmUXMBVSrbP

**Comment by mf-code** [comment]
[6108] - ScOtesQem83H

**Comment by mf-code** [comment]
[6108] - xmqPCZMhZ6LX

**Comment by mf-code** [comment]
[6108] - Lx0m62ecCYGX

**Comment by mf-code** [comment]
[6108] - 3erG3CpWm4C6

**Comment by mf-code** [comment]
[6108] - VO5JvXk4ewZQ

**Comment by mf-code** [comment]
[6108] - ajdFqlNkONrC

**Comment by mf-code** [comment]
[6108] - AW669hRlKkTZ

**Comment by mf-code** [comment]
[6108] - CnrDlKKWhDYp

**Comment by mf-code** [comment]
[6108] - F2nd1kgpxknv

**Comment by mf-code** [comment]
[6108] - GY99M14mzCSd

**Comment by mf-code** [comment]
[6108] - 7DyG5xfXGAL7

**Comment by mf-code** [comment]
[6108] - openjdk8 - Allure report on Netlify: https://5f60c7ad56e83f273363d2f6--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[6108] - openjdk10 - Allure report on Netlify: https://5f60c8639054b026f247236f--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[6112] - 4OU553tcvzqn

**Comment by mf-code** [comment]
[6112] - pIQ5BSJH4vZz

**Comment by mf-code** [comment]
[6112] - VA2KjqMscy69

**Comment by mf-code** [comment]
[6112] - SqV1SqXYtAzx

**Comment by mf-code** [comment]
[6112] - mktcvZEEtI3z

**Comment by mf-code** [comment]
[6112] - hhIa2ludMs5x

**Comment by mf-code** [comment]
[6112] - mcahD8LeGkvw

**Comment by mf-code** [comment]
[6112] -

**Comment by mf-code** [comment]
[6112] - F7WOP1jPJF13

**Comment by mf-code** [comment]
[6112] - yt6pMHZw2mVh

**Comment by mf-code** [comment]
[6112] - SuVHJXSbRNGX

**Comment by mf-code** [comment]
[6112] - CUL4XRGYNYvn

**Comment by mf-code** [comment]
[6112] - HhXSyESAlenc

**Comment by mf-code** [comment]
[6112] - T2nZ1oGk2aw9

**Comment by mf-code** [comment]
[6112] - urvtpit0Txa2

**Comment by mf-code** [comment]
[6112] - PgP79kiIYHYK

**Comment by mf-code** [comment]
[6112] - plidtcYj4L6d

**Comment by mf-code** [comment]
[6112] - oOBg7VGJgr4U

**Comment by mf-code** [comment]
[6112] - openjdk8 - Allure report on Netlify: https://5f6100946595f210715e70a4--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[6112] - openjdk10 - Allure report on Netlify: https://5f6101516ea1b5011a18682a--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[6114] - lY8a2ZHtFEK4

**Comment by mf-code** [comment]
[6114] - DhZ4mR54bczz

**Comment by mf-code** [comment]
[6114] - YjPIJhUaMXt3

**Comment by mf-code** [comment]
[6114] - oZylVLoj9Ig4

**Comment by mf-code** [comment]
[6114] - 42mUJma9QT4Q

**Comment by mf-code** [comment]
[6114] - iUJBhTqEHkNz

**Comment by mf-code** [comment]
[6114] - ywzwlE2uPmE9

**Comment by mf-code** [comment]
[6114] - 6ahjOkqscugv

**Comment by mf-code** [comment]
[6114] - gzbb9uxQeEwo

**Comment by mf-code** [comment]
[6114] - mI8W0VOkGO6I

**Comment by mf-code** [comment]
[6114] - BNUPsmNjugeu

**Comment by mf-code** [comment]
[6114] - u80wnXum2oGB

**Comment by mf-code** [comment]
[6114] - 1RmMEnNWgwcT

**Comment by mf-code** [comment]
[6114] - XbWBVYjUdOw2

**Comment by mf-code** [comment]
[6114] - b0AdAaufKGCe

**Comment by mf-code** [comment]
[6114] - aG2IbF4666d0

**Comment by mf-code** [comment]
[6114] - ec7J66iTuR9T

**Comment by mf-code** [comment]
[6114] - CEtGJHXCv6pG

**Comment by mf-code** [comment]
[6114] - openjdk8 - Allure report on Netlify: https://5f611899981a0130ec981163--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[6114] - openjdk10 - Allure report on Netlify: https://5f61198698d1e6344f6549f7--musing-swartz-6fc060.netlify.app

**Review by gbuh — APPROVED** [review]

**Pull request merged**